app/controllers/HasCorrectGoodsController.scala

```diff
@@ -81,7 +81,6 @@ class HasCorrectGoodsController @Inject() (
       }
     }
 
-  // TODO - this is still not functional, it is just to create the url. Implement this properly
```

Comment:
have we implemented this properly?

Reply:
There is a separate refactoring ticket for this